content/200-orm/500-reference/200-prisma-cli-reference.mdx

```diff
@@ -334,18 +334,17 @@ generator client {
-| `--data-proxy` | No       | The `generate` command will generate Prisma Client for use with the [Data Proxy](/platform/classic-projects/data-proxy). Mutually exclusive with `--accelerate` and `--no-engine`.                                                                                              |
-| `--accelerate` | No       | The `generate` command will generate Prisma Client for use with [Accelerate](/accelerate). Mutually exclusive with `--data-proxy` and `--no-engine`. Available in Prisma 5.1.0 and later.                                                                                       |
-| `--no-engine`  | No       | The `generate` command will generate Prisma Client without an accompanied engine for use with [Data Proxy](/platform/classic-projects/data-proxy) or [Accelerate](/accelerate). Mutually exclusive with `--data-proxy` and `--accelerate`. Available in Prisma 5.2.0 and later. |
-| `--watch`      | No       | The `generate` command will continue to watch the `schema.prisma` file and re-generate Prisma Client on file changes.                                                                                                                                                           |
+| Option         | Required | Description                                                                                                                                                                                                  | Default |
+| -------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------- |
+| `--accelerate` | No       | The `generate` command will generate Prisma Client for use with [Prisma Accelerate](/accelerate). Mutually exclusive with `--no-engine`. Available in Prisma 5.1.0 and later.                                |
+| `--no-engine`  | No       | The `generate` command will generate Prisma Client without an accompanied engine for use with [Prisma Accelerate](/accelerate). Mutually exclusive with `--accelerate`. Available in Prisma 5.2.0 and later. |
+| `--watch`      | No       | The `generate` command will continue to watch the `schema.prisma` file and re-generate Prisma Client on file changes.                                                                                        |
 
 <Admonition type="warning">
 
 **Deprecation Warning**
 
-As of Prisma 5.2.0, `--data-proxy` and `--accelerate` are deprecated in favor of `--no-engine` as Prisma Client no longer requires an option to work with Accelerate or Data Proxy. All options are available and work similarly, but we recommend `--no-engine` as it prevents an engine from being downloaded which will greatly impact the size of apps deployed to serverless and edge functions.
```

Comment:
These flags still exist so I wouldn't remove the note that they are deprecated.

Reply:
What's the point of still keeping around information of the `--data-proxy` flag though if Data Proxy doesn't exist any more? I don't see value in keeping it around 🤔

Reply:
> What's the point of still keeping around information of the --data-proxy flag though if Data Proxy doesn't exist any more? I don't see value in keeping it around 🤔

I'm not entirely certain, but if the --data-proxy flag was introduced in version 4.16.1 and the --accelerate flag followed later in version 5.0, then retaining both flags would be logical.

Reply:
Ah fair enough, I forgot that the flag was used in earlier Prisma versions for Accelerate as well. We're also still keeping this note in the "Getting started" section:

<img width="545" alt="image" src="https://github.com/prisma/docs/assets/4058327/52b6a317-76e6-4fa6-91f1-6b01abc119a9">

In that case, I think you're right and we should keep it.

Reply:
I reverted the change here and also added `--data-proxy` back to the table of options for `prisma generate` with a note that it should be used prior to 5.0.0.

Thanks for catching this and helping me figure this out to you both 🙏